UUID break old code (#1523)
ydb-platform/ydb-go-sdk: CHANGELOG.md

* BREAK OLD STYLE WORK WITH UUID. See https://github.com/ydb-platform/ydb-go-sdk/issues/1501 for details.

>   At the version you must explicit choose way for work with uuid: old with bug or new (fixed).

Reviewer: ```suggestion
  This version, you must explicit choose way to work with uuid: old, with bug, or new (fixed).
```

## v3.86.0
* Add workaround for bug in uuid send/receive from server. It is migration version. All native code and most database sql code worked with uuid continue to work.

> Dedicated version for migrate code for workaround/fix uuid bug. See https://github.com/ydb-platform/ydb-go-sdk/issues/1501 for details.

Reviewer: ```suggestion
Dedicated version to enable code migration from buggy way of sending UUID data to server to correct implementation. See https://github.com/ydb-platform/ydb-go-sdk/issues/1501 for details.
```
